bot/cogs/clean.py

```diff
@@ -168,35 +176,57 @@ def predicate_regex(message: Message) -> bool:
             channel_id=Channels.modlog,
         )
 
-    @group(invoke_without_command=True, name="clean", hidden=True)
+    @group(invoke_without_command=True, name="clean", aliases=["purge"])
```

Comment:
I know you didn't touch this, but while we're here, is there any particular reason why `invoke_without_command=True` is set? If it is left to the default `False`, the `@with_role` check can be on the command group, but not any of the subcommands - it will invoke the group command (and all checks) when any subcommand is called. The only thing that would need to change is to only send the help if there's no subcommand (and do a manual check) - 

```py
if not ctx.invoked_subcommand:
   await ctx.invoke(self.bot.get_command("help"), "clean")
```

Just a thought...